app/models/concerns/issued_at.rb

```diff
@@ -3,6 +3,7 @@ module IssuedAt
 
   included do
     before_create :initialize_issued_at
+    scope :by_issued_at, ->(issued_at) { where(issued_at: issued_at.beginning_of_month..issued_at.end_of_month) }
```

Comment:
unrelated?

Reply:
ah! this was a rubocop issue. This was originally multiple lines and I changed it to a single line.